src/modules/ui.js

```diff
@@ -3,29 +3,37 @@ import Leaderboard from './leaderboard.js';
 const myLeaderBoard = new Leaderboard();
 
 const manageUI = () => {
-  myLeaderBoard.display();
   const form = document.querySelector('#form');
   const scoresList = document.querySelector('#scores__list');
+  const message = document.querySelector('#message');
   const Refresh = document.querySelector('#Refresh');
   Refresh.addEventListener('click', () => {
     myLeaderBoard.display();
   });
 
-  form.addEventListener('click', (ev) => {
+  form.addEventListener('click', async (ev) => {
```

Comment:
- [ ] [OPTIONAL] Kindly use the `submit` event instead of `click` because currently, it's firing the event when clicking on any part of the form. I think it would be better for the event to get fired on clicking the submit button or hitting the Enter key. 😎️

Reply:
Thank you @amejid . I will fix it.🙏🏾